kong-3.7.0-0.rockspec

```diff
@@ -101,6 +101,10 @@ build = {
 
     ["kong.resty.dns.client"] = "kong/resty/dns/client.lua",
     ["kong.resty.dns.utils"] = "kong/resty/dns/utils.lua",
+
+    ["kong.resty.dns_client"] = "kong/resty/dns_client/init.lua",
+    ["kong.resty.dns_client.utils"] = "kong/resty/dns_client/utils.lua",
```

Comment:
the old lib is `kong.resty.dns.client` this one has `kong.resty.dns.dns_client`.

The `resty` part is a remnant of when the lib was first integrated into the Kong code base. So I agree with @bungle that it would be cleaner to move it. My preference would be moving it into `kong.dns.client`, which also solves the `-` vs `_` that @bungle brought up.

Reply:
also; please do not change the file names in the rockspec (we're now moving `init.lua` to `dns_client.lua`), such that we use the same names in code as well as in deployments. Changing the names is very annoying when debugging stack traces.

Reply:
resolved, move it into kong/dns/*.lua(client.lua and utils.lua)